src/main/java/com/minecolonies/structures/client/StructureClientHandler.java

```diff
@@ -0,0 +1,47 @@
+package com.minecolonies.structures.client;
+
+import com.minecolonies.structures.helpers.Settings;
+import com.minecolonies.structures.helpers.Structure;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.renderer.Vector3d;
+import net.minecraft.entity.Entity;
+import net.minecraft.util.math.BlockPos;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public final class StructureClientHandler
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1118)